centos.sh

```diff
@@ -102,49 +104,49 @@ generate_new_ramdisk() {
     # pick the latest kernel
     VERSION="$(ls -r -1 $FOLD/hdd/boot/vmlinuz-* | head -n1 |cut -d '-' -f 2-)"
 
-    if [ $IMG_VERSION -lt 60 ] ; then
+    if [ "$IMG_VERSION" -lt 60 ] ; then
       MODULESFILE="$FOLD/hdd/etc/modprobe.conf"
```

Comment:
this is an environment variable, exportet in functions.sh, I don't think that there is a suitable version to make this readonly? if you are aware of some method let me know and I will create an issue for it.